### PR TITLE
Improve HttpClient, Add selenium settings

### PR DESCRIPTION
**Pull request by RuslanPr0g**

### CODE REVIEW

File: src/Aurora.Presentation/libman.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1.0",
+  "defaultProvider": "cdnjs",
+  "libraries": []
```
Comment (GermanGolota) [inline src/Aurora.Presentation/libman.json:4]: Why do we need this file if there are no libraris?

File: src/Aurora.Infrastructure/Scrapers/XvideosScraper.cs [hunk 2]
```diff
@@ -95,7 +95,7 @@ public XvideosScraper(ILogger<XvideosScraper> logger, IWebClientService clientPr
                     var currentLinkAttributes = videoLinkNode.Attributes;
                     var videoLink = currentLinkAttributes["href"]?.Value;
                     if (videoLink is not null && videoLink.Contains("video") && !videoLink.Contains("videos"))
-                        searchVideoItem.SearchItemUrl = $"{_baseUrl}{videoLink}";
+                        searchVideoItem.SearchItemUrl = $"{client.BaseAddress}{videoLink}";
```
Comment (RuslanPr0g) [inline src/Aurora.Infrastructure/Scrapers/XvideosScraper.cs:98]: **here are "//" 2 slashes! needs to be fixed**

File: src/Aurora.Presentation/libman.json (hunk 1 above)
Comment (RuslanPr0g) [reply src/Aurora.Presentation/libman.json:4]: just in case (YAGNI)

File: src/Aurora.Infrastructure/Scrapers/PornhubScraper.cs [hunk 3]
```diff
@@ -105,7 +106,7 @@ private async Task<List<SearchItem>> ScrapVideos(string searchTerm, int maxNumbe
                     }
 
                     var currentLinkAttributes = videoLinkNode.Attributes;
-                    searchVideoItem.SearchItemUrl = $"{_baseUrl}{currentLinkAttributes["href"].Value}";
+                    searchVideoItem.SearchItemUrl = $"{client.BaseAddress}{currentLinkAttributes["href"].Value}";
```
Comment (RuslanPr0g) [inline src/Aurora.Infrastructure/Scrapers/PornhubScraper.cs:109]: fix the 2 slashes issue

File: src/Aurora.Presentation/libman.json (hunk 1 above)
Comment (GermanGolota) [reply src/Aurora.Presentation/libman.json:4]: sure thing then, you have business approval

**Pull request closed without merging**